src/styles/blocks/header.scss

```diff
@@ -27,9 +27,15 @@
     list-style: none;
     line-height: 60px;
 
+    transition: color 0.3s;
```

Comment:
Here you use the same duration of the transform, as in catalog.scss. We can use a variable. But it's just a recommendation.